sdk/src/transaction_builder.rs

```diff
@@ -95,7 +95,7 @@ impl TransactionFactory {
             gas_currency: Currency::XUS,
             transaction_expiration_time: 100,
             chain_id,
-            diem_version: 0,
+            diem_version: 2,
```

Comment:
just double check if I recalled my memory correctly, diem_version = 2 which enable the fn script?

Reply:
yep exactly